test/cypress/e2e/login.spec.js

```diff
@@ -72,23 +90,37 @@ describe('Login Page', () => {
     cy.get(ss.headerBalance).should('have.text', castNumberToBalanceString(accounts['testnet guy'].balance));
   });
 
+  /**
+   * If the switch network is set to true, devnet is chosen and custom node is specified,
+   * sign in happens to testnet
+   * @expect address matches
+   * @expect balance's three leading numbers match specified in constants
+   * @expect network status shows devnet
+   */
   it('Log in to Devnet', () => {
     cy.addLocalStorage('settings', 'showNetwork', true);
     cy.visit('/');
     chooseNetwork('dev');
     loginUI(accounts.genesis.passphrase);
     cy.get(ss.networkStatus).contains('Connected to devnet');
     cy.get(ss.headerAddress).should('have.text', accounts.genesis.address);
-    cy.wait(3000); // TODO Remove and improve after 1391 fix
     cy.get(ss.headerBalance).should('contain', castNumberToBalanceString(accounts.genesis.balance).substring(0, 3));
     cy.get(ss.nodeAddress).contains(networks.devnet.node);
   });
 
+  /**
+   * ?showNetwork parameter makes network switcher available
+   * @expect network switcher is visible
+   */
```

Comment:
I think you missed a typo **?**showNetwork

Reply:
I think it's on purpose as url param